src/Perpetuum.RequestHandlers/Characters/CharacterCreate.cs

```diff
@@ -19,6 +21,8 @@ namespace Perpetuum.RequestHandlers.Characters
 {
     public class CharacterCreate : IRequestHandler
     {
+        private const int WAIT_TIME_BEFORE_SENDING_MAIL_MS = 10000;
```

Comment:
Does this delay (10 seconds) seems reasonable?
With the tests I've done, the user will see "New Mail" notification a few seconds after the welcome window shows up.

Reply:
This is fine - only comment would be if the Task.delay() could use a TimeSpan instead
https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.delay?view=netframework-4.7.2#System_Threading_Tasks_Task_Delay_System_TimeSpan_System_Threading_CancellationToken_

Then something like this will be more maintainable and easier to tweak:
`TimeSpan.fromSeconds(10);` 
Only tradeoff is that you can't declare it as const I think.  But TimeSpan is the go-to object for time related things like delays on this codebase.